.travis.yml

```diff
@@ -2,14 +2,28 @@ language: python
 sudo: false
 os:
   - linux
+
 python:
   - 3.5
   - 3.6
-  - 3.6-dev
+  - 3.7-dev
+
+env:
+  - NUMPY=1.13
+  - NUMPY=1.14
+
+matrix:
+  allow_failures:
+    - python: "3.7-dev"
```

Comment:
Currently failing due to our dependencies (numba) not supporting it. Can take it out of allowed failures once all our dependencies support 3.7

Reply:
yea, agreed - nice one, thanks